### PR TITLE
docs(messaging): add additional APNS information

### PR DESCRIPTION
**Pull request by forbesgillikin**
Added additional information to account for latest APNs requirements

### Checklist

- I read the [Contributor Guide](/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
  - [x] Yes
- My change supports the following platforms;
  - [ ] `Android`
  - [x] `iOS`
- My change includes tests;
  - [ ] `e2e` tests added or updated in `packages/\*\*/e2e`
  - [ ] `jest` tests added or updated in `packages/\*\*/__tests__`
- [ ] I have updated TypeScript types that are affected by my change.
- This is a breaking change;
  - [ ] Yes
  - [x] No

### Test Plan

---

Think `react-native-firebase` is great? Please consider supporting the project with any of the below:

- 👉 Star this repo on GitHub ⭐️
- 👉 Follow [`React Native Firebase`](https://twitter.com/rnfirebase) and [`Invertase`](https://twitter.com/invertaseio) on Twitter

### CODE REVIEW

**Comment by forbesgillikin** [comment]
@mikehardy great! This is my first ever contribution to any open source project 😄 . Glad it's a good one.

File: docs/messaging/usage/index.md [hunk 1]
```diff
@@ -208,6 +208,34 @@ admin.messaging().sendToDevice(
...
+    apns: {
+      payload: {
+        aps: {
+          contentAvailable: true
+        }
+      },
+      headers: {
+        'apns-push-type': 'background',
+        'apns-priority': '5',
+        'apns-topic': //app bundle identifier
```
Comment (Salakar) [inline docs/messaging/usage/index.md:227]:
```suggestion
        'apns-topic': '' // your app bundle identifier
```

File: docs/messaging/usage/index.md [hunk 2]
```diff
@@ -208,6 +208,34 @@ admin.messaging().sendToDevice(
 );
 ```
 
+For iOS specific "data-only" messages, the message must include the approriate APNs headers as well as the `content-available` flag in order to trigger the background handler. For example, if using the Node.js [`firebase-admin`](https://www.npmjs.com/package/firebase-admin) package to send a "data-only" message to an iOS device:
```
Comment (Salakar) [inline docs/messaging/usage/index.md:211]:
```suggestion
For iOS specific "data-only" messages, the message must include the appropriate APNs headers as well as the `content-available` flag in order to trigger the background handler. For example, if using the Node.js [`firebase-admin`](https://www.npmjs.com/package/firebase-admin) package to send a "data-only" message to an iOS device:
```

**Comment by Salakar** [comment]
Thanks for this, shipped 🚢

**Pull request merged**